pkgs/by-name/ch/choose-gui/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  xcbuild,
+  coreutils,
+  darwin,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "choose-gui";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "chipsenkbeil";
+    repo = "choose";
+    rev = version;
+    hash = "sha256-oR0GgMinKcBHaZWdE7O+mdbiLKKjkweECKbi80bjW+c=";
+  };
+
+  buildInputs = [
+    coreutils
+    darwin.apple_sdk.frameworks.Cocoa
+    xcbuild
+  ];
+
+  buildPhase =
+    let
+      arch =
+        with stdenv.hostPlatform;
+        if isx86_64 then
+          "x86_64"
+        else if isAarch64 then
+          "arm64"
+        else
+          throw "${pname}-${version} does not support ${parsed.cpu.arch} architecture";
+    in
```

Comment:
Can you move this `let arch = ...; in` to line 10 (before `stdenv.mkDerivation rec {`)?

I understand you wanted to keep it scoped right. But in nixpkgs this is unusual, these kind of 'let .. in' are usually kept before `stdenv.mkDerivation`.
To let the formatter format it right, I suggest you make it a single line and the formatter adjust it for you. The formatter is not optionated on break lines.